docs/scheduler/filter.md

```diff
@@ -11,22 +11,99 @@ weight=4
 
 # Filters
 
-The `Docker Swarm` scheduler comes with multiple filters.
+Filters tell Docker Swarm which nodes to use when creating and running a
+container.
 
-The following filters are currently used to schedule containers on a subset of nodes:
+The Docker Swarm scheduler provides the following filters:
 
-* [Constraint](#constraint-filter)
-* [Affinity](#affinity-filter)
-* [Port](#port-filter)
-* [Dependency](#dependency-filter)
-* [Health](#health-filter)
+* [Node constraint](#node-constraint-filter)
+* [Container affinity](#container-affinity-filter)
+* [Node port](#node-port-filter)
+* [Container dependency](#container-dependency-filter)
+* [Node health](#node-health-filter)
 
-You can choose the filter(s) you want to use with the `--filter` flag of `swarm manage`
+You can choose the filter(s) you want to use with the `--filter` flag of
+`swarm manage`. By default all filters are enabled.
 
-## Constraint Filter
+> *Note*: filters that operate on container configuration
```

Comment:
The two other filters operate on node labels and node health, see "Node constraint filter" and "Node health filter".